config/application.rb

```diff
@@ -11,6 +11,9 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
 
+    # Enable exceptions override
+    config.exceptions_app = self.routes
```

Comment:
I'm not sure if you need this or not. If this doesn't work without it then I'm happy with this.